### PR TITLE
feat: add the "outdated" command

### PR DESCRIPTION
**Pull request by ss141309**
I was unable to check the output of this command for the brew and choco managers, therefore if anything is wrong with them, please mention it here so that I can fix it

### CODE REVIEW

**Comment by dilawar** [comment]
Is it possible to fix the failing linux tests? I know its not failing due to this MR.

**Comment by ss141309** [comment]
it seems that the tests fail due to a race condition on linux. We can fix that either by running cargo test with a single thread on our github workflow (by using `cargo test -- --test-threads=1`) or by using the `serial_test` crate, which essentially does the same thing.
Which one do you prefer?

**Comment by dilawar** [comment]
`test-threads=1` would be better

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Build
         run: cargo build --verbose
       - name: Run tests
-        run: cargo test --all-features --verbose
+        run: cargo test --all-features --verbose -- --test-threads=1
```
Comment (dilawar) [inline .github/workflows/rust.yml:30]: Add a comment here so others don't remove it. Or just mentioned the note.

**Review by dilawar — APPROVED** [review]
If something breaks on OSX/Windows, I'll open a ticket.

**Pull request merged**